docker/slim-stable/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Sysdig <support@sysdig.com>"
```

Comment:
I think we want a different name here.

Reply:
```suggestion
LABEL maintainer="opensource@sysdig.com"
```

Reply:
I don't think we want an at Sysdig

Reply:
Atm we only have the new email "opensource@sysdig.com" (which is the same I used when refactored the falco-builder and falco-tester docker images).

But I agree with you that @falco.org would be way better! :)